### ISSUE TITLE
Production Stable release schedule?

### ISSUE THREAD

**Issue by AyushG3112** [vuikit/vuikit#74]
Hi,

Any estimate when the UIKit 3 compatible version will be considered production-stable?

Thanks and regards
Ayush

**Comment by miljan-aleksic** [comment]
Hi Ayush, I am working on the documentation right now and fixing some edge situations. I estimate one more week :)

**Comment by AyushG3112** [comment]
Great thanks! :)

On 07-Mar-2018 2:33 PM, "Miljan" <notifications@github.com> wrote:

Hi Ayush, I am working on the documentation right now and fixing some edge
situations. I estimate one more week :)

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/vuikit/vuikit/issues/74#issuecomment-371070627>, or mute
the thread

**Comment by miljan-aleksic** [comment]
Yes! It was announced on Tweeter. The 0.8 version is only UIkit 3 compatible, for UIkit 2 you must use the 0.6 which is considered deprecated, though.

**Comment by AyushG3112** [comment]
Our UI is developed in UIKit 2 so I've got no choice. Is the v0.6 still available on NPM though?

**Comment by miljan-aleksic** [comment]
Yes, sure. Previous versions are available - https://unpkg.com/vuikit@0.6.3/
